### PR TITLE
*: replace `file(s)` syntax and add `path/to/` to more things

### PR DESCRIPTION
**Pull request by pixelcmtd**
Breakout # 3 for #9578. I actually intended for this to just be a `file(s)` thing, but I just found so many more cases of missing `path/to/`.

```sh
fd -t f . pages | xe -j20 -- -i {} sed 's/{{\([\/a-zA-Z_]*file\)(s)}}/{{\11 \12 ...}}/g;s/{{filename}}/{{path\/to\/file}}/g;s/{{file1 file2 ...}}/{{path\/to\/file1 path\/to\/file2 ...}}/g
```

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.it/common/rm.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 - Rimuovi file interattivamente, chiedendo conferma prima di rimuovere ogni file:
 
-`rm -i {{file(s)}}`
+`rm -i {{file1 file2 ...}}`
```
Comment (EmilySeville7cfg) [inline pages.it/common/rm.md:20]:
```suggestion
`rm -i {{path/to/file1 path/to/file2 ...}}`
```

Mandatory change: [path format is wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): `path/to/` is expected but `file` is met.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- One or more mandatory changes detected, changes requested.
- Thanks for a contribution! ❤️

File: pages.it/common/rm.md (hunk 1 above)
Comment (pixelcmtd) [reply pages.it/common/rm.md:20]: As far as I know, `path/to/` is not correct Italian.

Comment (kbdharun) [reply pages.it/common/rm.md:20]: > As far as I know, `path/to/` is not correct Italian.

Yeah, I think we can use the one from common arguments.

Comment (EmilySeville7cfg) [reply pages.it/common/rm.md:20]: Suggestion updated according to google translate.

**Review by EmilySeville7cfg — APPROVED** [review]

**Pull request merged**